mock/py/mockbuild/buildroot.py

```diff
@@ -207,6 +207,13 @@ def doChroot(self, command, nosync=False, *args, **kargs):
             self.uid_manager.restorePrivs()
         return result
 
+    def all_chroot_packages(self):
+        """package set, result of rpm -qa in the buildroot"""
+        out = util.do([self.config['rpm_command'], "-qa",
+                       "--root", self.make_chroot_path()],
+                      returnOutput=True, printOutput=False, shell=False)
```

Comment:
I've added this in the second commit. Can squash or unpush it, whatever the maintainers prefer. It works.